apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs

```diff
@@ -244,6 +244,73 @@ public partial class BigtableClient
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Settings for creating <see cref="BigtableServiceApiClient"/>s.
+        /// </summary>
+        public sealed class ClientCreationSettings
+        {
+            /// <summary>
+            /// Instantiate with the specified settings.
+            /// </summary>
+            /// <param name="clientCount">Optional.
```

Comment:
Perhaps should add here
If not set will defaults to count of processors

Reply:
Yeah, that's probably a good idea, although we may change that heuristic in the future, but we can change the docs at the same time too if that's the case.